src/table/components/HeadCellMenu.tsx

```diff
@@ -13,12 +13,24 @@ import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward';
 import MoreHoriz from '@mui/icons-material/MoreHoriz';
 import { StyledMenuIconButton, StyledCellMenu } from '../styles';
 import { GeneratedStyling } from '../types';
-import { ExtendedTranslator } from '../../types';
+import { ChangeSortOrder, Column, ExtendedTranslator } from '../../types';
 
-const ListItems = ({ children, itemTitle }: { children: any; itemTitle: string }) => {
+const ListItems = ({
+  children,
+  itemTitle,
+  column,
+  sortOrder,
+  changeSortOrder,
+}: {
+  children: any;
+  itemTitle: string;
+  column: Column;
+  sortOrder: string;
+  changeSortOrder: ChangeSortOrder;
+}) => {
   return (
     <ListItem disablePadding>
-      <ListItemButton>
+      <ListItemButton onClick={() => changeSortOrder(column, sortOrder)}>
```

Comment:
and this would be something like
```suggestion
      <ListItemButton onClick={(e: Event) => sortFromMenu(e, SortOrder)>
```